packages/mobile/src/account/reducer.ts

```diff
@@ -61,7 +60,6 @@ export const initialState = {
   backupCompleted: false,
   backupDelayedTime: 0,
   socialBackupCompleted: false,
-  dismissedEarnRewards: false,
```

Comment:
Should this also be removed from the `schema` file?

Reply:
Good catch!